packages/block-library/src/group/edit.js

```diff
@@ -67,6 +68,16 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 		}
 	);
 
+	const { __unstableMarkNextChangeAsNotPersistent } =
+		useDispatch( blockEditorStore );
+	const { inherit = false } = layout;
+	useEffect( () => {
+		if ( inherit ) {
+			__unstableMarkNextChangeAsNotPersistent();
+			setAttributes( { layout: { inherit } } );
```

Comment:
I'm probably doing something wrong elsewhere, but `{ layout: { inherit } }` doesn't actually get saved to the block attributes without this bit of code. So without it, if you add a Group and then go into code view you can see the layout attribute isn't there. I feel there must be something I'm missing, not sure what 😅

Reply:
Ah, gotcha! I wonder if it's to do with the behaviour where a default value isn't serialized, but if we switch a value off and on so that `setAttributes` is explicitly called, then it _is_ serialized 🤔

Reply:
That might be it! I'll dig around for any precedent with adding default attributes to blocks and see if there's a better way.

Reply:
Looking through `block-library`, I can't find another instance of an attribute having to be set on a block by default, without being tied to a specific action (e.g. in File and Image, new attributes have been added, but they are only set during upload). Not sure we'll be able to do without explicitly setting the attribute.

This is awkward, especially if new blocks want to have content width set by default 🤔 

I'll continue looking into it.